DEBUG 2018-06-29 16:07:20,244: 	 Executing command: 'submit'
DEBUG 2018-06-29 16:07:20,245: 	 Found CRAB cache file /homeui/hoh/.crab3
DEBUG 2018-06-29 16:07:20,255: 	 Checking credentials
DEBUG 2018-06-29 16:07:20,255: 	 Found proxy file /tmp/x509up_u761
DEBUG 2018-06-29 16:07:20,256: 	 Getting proxy life time left
DEBUG 2018-06-29 16:07:20,860: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u761 -timeleft
 output : 604787

 error: 
 retcode : 0
DEBUG 2018-06-29 16:07:21,465: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u761 -actimeleft
 output : 604786

 error: 
 retcode : 0
DEBUG 2018-06-29 16:07:21,465: 	 Proxy valid for 167:59:47 hours
INFO 2018-06-29 16:07:21,465: 	 Will use CRAB configuration file crab_cfg.py
DEBUG 2018-06-29 16:07:21,465: 	 Loading CRAB configuration file.
DEBUG 2018-06-29 16:07:21,484: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
ERROR 2018-06-29 16:07:21,484: 	 Invalid CRAB configuration: A local DBS instance 'phys03' was specified for reading an input dataset of tier NANOAODSIM.
Datasets of tier different than USER must be read from the global DBS instance; this is, set Data.inputDBS = 'global'.
If Data.inputDBS would not be specified, the default 'global' ('https://cmsweb.cern.ch/dbs/prod/global/DBSReader') would be used.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2018-06-29 16:07:21,484: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1805.patch1/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1805.patch1/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1805.patch1/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1805.patch1/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 303, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1805.patch1/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 89, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: A local DBS instance 'phys03' was specified for reading an input dataset of tier NANOAODSIM.
Datasets of tier different than USER must be read from the global DBS instance; this is, set Data.inputDBS = 'global'.
If Data.inputDBS would not be specified, the default 'global' ('https://cmsweb.cern.ch/dbs/prod/global/DBSReader') would be used.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
